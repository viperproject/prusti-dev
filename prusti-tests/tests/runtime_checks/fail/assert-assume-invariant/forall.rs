//@run: 101
use prusti_contracts::*;
#[trusted]
fn main() {
    let v = vec![1,2,3,4,5,6];
    prusti_assume!(#[insert_runtime_check]forall(|x: usize| x < 6 ==> v[x] < 6));
}
