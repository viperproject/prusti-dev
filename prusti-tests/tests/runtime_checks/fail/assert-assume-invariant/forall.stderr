thread 'main' panicked at $DIR/forall.rs:6:43:
Prusti Runtime Checks: Contract prusti_assume!(forall(|x: usize| x < 6 ==> v[x] < 6)) was violated at runtime
	> expression x < 6 ==> v[x] < 6 was violated for index x=5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
